examples/react-apollo-live/README.md

```diff
@@ -0,0 +1,13 @@
+# Grafbook: Server sent events (SSE) x Clerk ⨯ React
+
+## Getting Started
+
+1. Run `npx degit grafbase/grafbase/examples/react-clerk-live grafbase-with-react-clerk-live` to clone this example
+1. Change directory into the new folder `cd grafbase-with-react-clerk-live`
+1. Run `cp .env.example .env` to copy the example `.env.example` file to `.env` in the root of your project
+1. Open `.env` in your code editor, and provide your Grafbase API endpoint and [Clerk frontend and backend API keys](https://dashboard.clerk.dev/last-active?path=api-keys)
```

Comment:
We could remove this step entirely for demo purposes if we used the JWT provider. Let me know what you think about the suggestion below in the review comments.

Reply:
done